sql/upgrade/vectors--0.1.12--0.1.13.sql

```diff
@@ -1,4 +1,4 @@
-\echo Use "ALTER EXTENSION vectors UPDATE TO '0.2.0'" to load this file. \quit
+\echo Use "ALTER EXTENSION vectors UPDATE TO '0.1.13'" to load this file. \quit
```

Comment:
It's not a typo. We do not really support update among versions `< 0.2.0` since update scripts are not included in these releases.